source/reference/minio-mc/mc-support-upload.rst

```diff
@@ -0,0 +1,93 @@
+======================
+``mc support upload``
+======================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 1
+
+.. mc:: mc support upload
+
+.. include:: /includes/common-mc-support.rst
+   :start-after: start-minio-only
+   :end-before: end-minio-only
+
+Description
+-----------
+
+.. start-mc-support-upload-desc
+
+:mc:`mc support upload` copies a file from the local file system to a SUBNET ticket.
+
+.. end-mc-support-upload-desc
+
```

Comment:
```suggestion

.. include:: /includes/common-mc-support.rst
   :start-after: start-minio-only
:end-before: end-minio-only

```

Reply:
We had this above the heading. But i moved it to here.